test/e2e_source_map_test.ts

```diff
@@ -22,9 +22,6 @@ import {createOutputRetainingCompilerHost, createSourceReplacingCompilerHost, to
 
 import * as testSupport from './test_support';
 
-describe('source maps with TsickleCompilerHost', () => {
```

Comment:
Unfortunately, the tests I added don't pass with the baseline TsickleCompilerHost code path.  This seemed like the least confusing low effort option.  Do you think it's worth eg splitting them into their own file?  Can you think of an easy way to make them only run for the transformer code path without adding another set of flags?